kinto/config/kinto.tpl

```diff
@@ -82,22 +237,43 @@ kinto.includes = kinto.plugins.default_bucket
 # kinto.blog_article_record_cache_expires_seconds = 3600
 
 #
+# Project information
+#
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#project-information
+#
+# kinto.version_json_path = ./version.json
+# kinto.error_info_link = https://github.com/kinto/kinto/issues/
+# kinto.project_docs = https://kinto.readthedocs.io
+# kinto.project_version = {project_version}
+# kinto.version_prefix_redirect_enabled = true
+
+#
+# Application Profiling
+#
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#application-profiling
+#
+# kinto.profiler_enabled = true
+# kinto.profiler_dir = /tmp/profiling
+
+#
 # Production settings
```

Comment:
production settings should go relatively on top, since they are more useful than profiling etc.